addons/website_project_issue/controllers/main.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import werkzeug
+
+from openerp import http
+from openerp.http import request
+
+from openerp.addons.website_portal.controllers.main import website_account
+
+class WebsiteAccount(website_account):
+    @http.route(['/my', '/my/home'], type='http', auth="user", website=True)
+    def account(self):
+        response = super(WebsiteAccount, self).account()
+        project_issues = request.env['project.issue'].search([])
+        response.qcontext.update({
+            'issues': project_issues
+        })
+        return response
+
+class website_project_issues(http.Controller):
+    @http.route(['/my/issues/<model("project.issue"):issue>'], type='http', auth="user", website=True)
+    def issues_followup(self, issue=None):
+        has_timesheet_enabled = issue._fields.get('timesheet_ids')
```

Comment:
Don't do anything Harshad. Here we do not want to add an explicit dependency to `project_issue_sheet` and the code already checks whether the field exists or not. (A cleaner way would require to add another bridge module `website_project_issue_sheet`, but that's too much for such a small feature)